bigframes/testing/utils.py

```diff
@@ -83,7 +90,7 @@ def assert_series_equivalent(pd_series: pd.Series, bf_series: bpd.Series, **kwar
 
 
 def _normalize_all_nulls(col: pd.Series) -> pd.Series:
-    if col.dtype == bigframes.dtypes.FLOAT_DTYPE:
+    if col.dtype in (bigframes.dtypes.FLOAT_DTYPE, bigframes.dtypes.INT_DTYPE):
```

Comment:
[no action required] Pandas's mistreatment of the so-called nullable types does make me want to switch to pyarrow types sooner rather than later. Let's refresh our 3.0 plans soon.